GameData/SPEngine/Parts/Gamma_proc.cfg

```diff
@@ -45,7 +41,7 @@
 	}
 }
 
-+PART[ROE-RD0110]:NEEDS[ROEngines]:AFTER[ZREALPLUME]
++PART[ROE-RD0110Vernier]:NEEDS[ROEngines]:AFTER[ZREALPLUME]
```

Comment:
Since this requires ROEngines#69, the non-Alternate model (SPEngineGamma, line 1) should not be `:NEEDS[!ROEngines]`.  Otherwise players with an unpatched ROEngines won't have an X-class part at all; and two is better than none.